apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -0,0 +1,237 @@
+          no matter what opportunities they pursue in the future.
+        </p>
+        <p>
+          Please let me know if you have any questions and thank you for your
+          continued support of your child and of our classroom!
+        </p>
+        <p>{teacherName}</p>
+      </article>
+    </div>
+  );
+}
+ParentLetter.propTypes = {
+  loginType: PropTypes.oneOf(Object.values(SectionLoginType)).isRequired,
+  sectionCode: PropTypes.string,
+  teacherName: PropTypes.string.isRequired
+};
+
+const Header = () => {
+  return (
+    <header style={{backgroundColor: color.teal, marginBottom: 30}}>
+      <img
+        src="/shared/images/CodeLogo_White.png"
+        style={{
+          height: 30,
+          margin: 15
+        }}
+      />
+    </header>
+  );
+};
+
+const SignInInstructions = ({loginType, sectionCode}) => {
+  let steps;
+  switch (loginType) {
+    case SectionLoginType.clever:
+      steps = (
+        <ol>
+          <li>
+            Have your students log in to their Clever account at{' '}
+            <a href="https://www.clever.com/">www.clever.com</a> (click "Sign in
+            as a student" at the top right)
+          </li>
+          <li>
+            Click on the Code.org logo on the Clever dashboard. The logo looks
+            like this:
+            <br />
+            <img
+              src="/shared/images/clever_code_org_logo.png"
+              style={{
+                width: 60,
+                margin: 10
+              }}
+            />
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.google_classroom:
+      steps = (
+        <ol>
+          <GoToSignIn />
+          <li>Choose 'Sign in with Google'</li>
+          <li>Sign in via the Google sign-in dialog</li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.picture:
+      steps = (
+        <ol>
+          <GoToSectionSignIn sectionCode={sectionCode} />
+          <li>Click on their picture password and then click 'Sign in'</li>
+          <li>
+            If your student does not remember their picture password, please
+            email me and I will provide it
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.word:
+      steps = (
+        <ol>
+          <GoToSectionSignIn sectionCode={sectionCode} />
+          <li>Type in their secret words and then click 'Sign in'</li>
+          <li>
+            If your student does not remember their password, please email me
+            and I will provide it
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.email:
+    default:
+      steps = (
+        <ol>
+          <GoToSignIn />
+          <li>
+            Have them enter their email and password and then click 'Sign in'
+          </li>
+          <li>
+            If your student does not remember their password, they can reset it
+            from the Sign in screen
```

Comment:
also nit - should 'In' be capitalized?

Reply:
Whoops! Pulled this as-is from the spec, but you're right, it's weird.  I'd actually lower "sign."  Our convention seems to be "Sign in" when quoting the button, and "sign in" everywhere else.